tests/test_intercept.js

```diff
@@ -3751,7 +3754,7 @@ function checkDuration(t, ms) {
     ended = true
     const fin = process.hrtime(start)
     const finMs =
-      fin[0] * 1e9 + // seconds -> ms
+      fin[0] * 1e3 + // seconds -> ms
```

Comment:
`fin[0]` is often 0, though when it isn't, this number is wrong. 1 second is 1e3 milliseconds.

Reply:
I wonder why not just write "1000" instead of `1e3` :D Great catch!

Reply:
That's fine; I guess I find them about equally understandable.